### PR TITLE
[unifi] Provide LED channel for access point

### PR DESCRIPTION
**Pull request by jlaur**
Adds new channels for controlling LED on access point

This is a simplified version replacing #17534 since color and brightness seems unsupported on all tested access points, even though exposed by the API (and setable):
```json
"led_override_color": "#0000ff",
"led_override": "default",
"led_override_color_brightness": 100,
```

`led_override` can be `on`, `off` or `default`. After using this channel, it will be either **on** or **off** and can not be set to **default** by the binding. This behavior is the same as in the UniFi app for Android. `default` means using the site-wide setting for LED.

### CODE REVIEW

File: bundles/org.openhab.binding.unifi/src/main/java/org/openhab/binding/unifi/internal/handler/UniFiAccessPointThingHandler.java [hunk 1]
```diff
@@ -152,6 +152,10 @@ protected State getChannelState(final UniFiDevice device, final String channelId
                     state = new QuantityType<>(device.getExperience(), Units.PERCENT);
                 }
                 break;
+            case CHANNEL_AP_LED:
+                String override = device.getLedOverride();
+                state = "default".equals(override) ? UnDefType.UNDEF : OnOffType.from("on".equals(override));
+                break;
```
Comment (lsiepel) [inline bundles/org.openhab.binding.unifi/src/main/java/org/openhab/binding/unifi/internal/handler/UniFiAccessPointThingHandler.java:158]: Very minor:
```suggestion
                state = "default".equals(override) ? UnDefType.UNDEF : OnOffType.from(override);
```

File: bundles/org.openhab.binding.unifi/src/main/resources/OH-INF/update/thing-updates.xml [hunk 2]
```diff
@@ -36,6 +36,13 @@
 				<description>The average experience of the connected clients</description>
 			</add-channel>
 		</instruction-set>
+		<instruction-set targetVersion="2">
+			<add-channel id="led">
+				<type>system:power</type>
+				<label>LED</label>
+				<description>Switches the LED on or off</description>
```
Comment (lsiepel) [inline bundles/org.openhab.binding.unifi/src/main/resources/OH-INF/update/thing-updates.xml:43]: Setting label and description should not be needed for new added channels. It won't harm, but just saying.

File: bundles/org.openhab.binding.unifi/src/main/java/org/openhab/binding/unifi/internal/handler/UniFiAccessPointThingHandler.java (hunk 1 above)
Comment (lsiepel) [reply bundles/org.openhab.binding.unifi/src/main/java/org/openhab/binding/unifi/internal/handler/UniFiAccessPointThingHandler.java:158]: Some how my vscode review plugin is one line off. obvious this comment is about line 157

Comment (jlaur) [reply bundles/org.openhab.binding.unifi/src/main/java/org/openhab/binding/unifi/internal/handler/UniFiAccessPointThingHandler.java:158]: Thanks, indeed that looks better.

**Comment by jlaur** [comment]
@DrRSatzteil - perhaps you'd like to have a look?

**Review by lsiepel — APPROVED** [review]
Thanks LGTM. Lets give some time for others to comment.

**Review by DrRSatzteil — COMMENTED** [review]
LGTM

**Pull request merged**